rpm/harbour-osmscout.spec

```diff
@@ -48,6 +48,17 @@ BuildRequires:  qt5-qttools-linguist
 %description
 OSM Scout is offline map viewer and routing application.
 
+PackageName: OSM Scout
+Type: desktop-application
+Categories:
+  - Maps
```

Comment:
With Pure Maps and mapping libs I have also added "Science" as it is corresponding Main category from https://specifications.freedesktop.org/menu-spec/latest/apa.html . However, as we have "Location and Navigation" section available for browsing directly (which is mapped to category Maps), that could probably be skipped now. My categories were done before GUI was ready.

Reply:
ok. I will omit Science